lib/rouge/lexers/cython.rb

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    load_lexer 'python.rb'
+
+    class Cython < Python
+      title "Cython"
+      desc "Cython and Pyrex source code (cython.org)"
+      tag 'cython'
+      aliases 'pyx', 'pyrex'
+      filenames '*.pyx', '*.pxd', '*.pxi'
+      mimetypes 'text/x-cython', 'application/x-cython'
+
+      def self.keywords
+        @keywords ||= super + Array.new(%w(
+          by ctypedef except? fused gil nogil 
+        ))
+      end
+
+      dotted_identifier = /[a-z_.][a-z0-9_.]*/i
+      prepend :root do
+        rule %r/(cimport)(\s+)(#{dotted_identifier})/ do
+          groups Keyword::Namespace, Text, Name::Namespace
+        end
+
+        rule %r/(from)((?:\\\s|\s)+)(#{dotted_identifier})((?:\\\s|\s)+)(cimport)/ do
+          groups Keyword::Namespace,
+                 Text,
+                 Name::Namespace,
+                 Text,
+                 Keyword::Namespace
+        end
+
+        rule %r/(cp?def)(\s+)/ do
+          groups Keyword, Text
+          push :cdef
+        end
+
+        rule %r/(cdef)(:)/ do
+          groups Keyword, Punctuation
+        end
+
+        rule %r/(struct)((?:\s|\\\s)+)/ do
```

Comment:
It's not a big deal but the order of `\s|\\\s` is reversed from earlier. I don't think it has any impact but it might be easier for future refactoring if it's consistent.

Reply:
Agreed, reversed for easier reading.